starfish/test/full_pipelines/api/test_dartfish.py

```diff
@@ -1,26 +1,24 @@
+import os
+import sys
+
 import numpy as np
 import pandas as pd
+from definitions import ROOT_DIR
 
-from starfish import Experiment, IntensityTable
-from starfish.image._filter.scale_by_percentile import ScaleByPercentile
-from starfish.image._filter.zero_by_channel_magnitude import ZeroByChannelMagnitude
-from starfish.spots._detector.pixel_spot_detector import PixelSpotDetector
+from starfish import IntensityTable
 from starfish.types import Features
 
+os.environ["USE_TEST_DATA"] = "1"
+sys.path.append(os.path.join(ROOT_DIR, "notebooks/py/"))
+dartfish = __import__('DARTFISH_Pipeline_-_Human_Occipital_Cortex_-_1_FOV')
+
 
 def test_dartfish_pipeline_cropped_data():
 
     # set random seed to errors provoked by optimization functions
     np.random.seed(777)
```

Comment:
I believe you need to set the random seed before you run the pipeline.

Consider just moving the __import__ statement into this method.